packages/components/src/range-control/styles/range-control-styles.js

```diff
@@ -232,16 +232,16 @@ const tooltipShow = ( { show } ) => {
 };
 
 const tooltipPosition = ( { position } ) => {
-	const isTop = position === 'top';
+	const isBottom = position === 'bottom';
 
-	if ( isTop ) {
+	if ( isBottom ) {
 		return css`
-			top: -80%;
+			bottom: -80%;
 		`;
 	}
 
 	return css`
-		bottom: -80%;
+		top: -80%;
```

Comment:
I'm not sure all of the changes here are needed. It looks to me like it should be possible to add a `position="bottom"` prop to `SimpleTooltip` here:
https://github.com/WordPress/gutenberg/blob/149502cc880594ba3608538c57145ac586d24bf3/packages/components/src/range-control/index.js#L258-L265

That gets passed through to `Tooltip` and then to the `useTooltipPosition` hook, and finally on to `tooltipPosition`.

Reply:
Right, this makes sense.